src/algorithmForNN/fileUtils.py

```diff
@@ -12,17 +13,34 @@ def get_point_cloud_from_bin_file(path_to_bin_file: str) -> o3d.geometry.PointCl
     return point_cloud
 
 
-def write_ones_to_file_by_index_list(
-    point_cloud_size: int, index_list: list, path_to_new_label_file: str
+def generate_labels_and_object_files(
+    point_cloud_size: int,
+    index_list: list,
+    path_to_new_label_file: str,
+    path_to_new_object_file: str,
 ):
     """
     Function changes those lines in the file, the point with the index of which belongs to any plane
     """
-    list_strings = ["0\n"] * point_cloud_size
+    labels_list = ["["] + ["0,"] * point_cloud_size + ["]"]
     for index in index_list:
-        list_strings[index] = "1\n"
-    with open(path_to_new_label_file, "w") as file:
-        file.writelines(list_strings)
+        labels_list[index + 1] = "1,"
+    labels_list[-2] = labels_list[-2][:-1]
+
+    labels_string = ""
+    for item in labels_list:
+        labels_string += item
+
+    with open(path_to_new_label_file, "wb") as label_file, open(
+        path_to_new_object_file, "wb"
+    ) as object_file:
+
+        compresses_labels_string = FIC.compress(LZW.compress(labels_string))
+        compressed_objects_string = FIC.compress(LZW.compress("[]"))
+        for byte in compresses_labels_string:
```

Comment:
Do we need this iteratior? Why not just smth like https://stackoverflow.com/a/18367068